src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java

```diff
@@ -236,4 +227,26 @@ private static boolean checkIsAllLinesAreWhitespace(String[] lines, int lineFrom
         }
         return result;
     }
+
+    /**
+     * Calculates the left curly corresponding to the block to be checked.
+     *
+     * @param ast a {@code DetailAST} value
+     * @return the left curly corresponding to the block to be checked
+     */
+    private static DetailAST findLeftCurly(DetailAST ast) {
+        final DetailAST leftCurly;
+        if (ast.getType() == TokenTypes.LITERAL_CASE
+                && ast.getNextSibling() != null
+                && "{".equals(ast.getNextSibling().getFirstChild().getText())) {
```

Comment:
@romani  Another option, lemme just confirm if it'd work properly:
replace `"{".equals(ast.getNextSibling().getFirstChild().getText()))` by 
`ast.getNextSibling().getFirstChild().getType() == TokenTypes.SLIST` 
Would that be alright ?